pinot-tools/src/main/resources/log4j2.xml

```diff
@@ -44,7 +44,7 @@
 
   </Appenders>
   <Loggers>
-    <Root level="info" additivity="false">
+    <Root level="warn" additivity="false">
```

Comment:
This seems unrelated to this PR? Would be good to call it out in the description, along with the motivation for the change.

Reply:
You are right. Reverted this since it was for local debugging purpose.